### DISCUSSION TITLE
Guids and DateTimes not being scrubbed

### DISCUSSION THREAD

**Discussion by cpneale** [VerifyTests/Verify#596] [Q&A] [answered]
Hi, 

Thanks for this project, I just started trying it out as I love the idea of it. 

Any idea what I'm doing wrong here?  One object is scrubbed of Dates and Guids, the other is not.

```cs
await Verify(new {_response, _createdNode});
```

gives the following '*.received.text'

```
{
    _response: {
        StatusCode: 201,
        Headers: {
            Date: [
                Sat, 13 Aug 2022 10:38:49 GMT
            ],
            Location: [
                http://localhost:5000/api/nodes/6c3bcecb-e899-4776-91c9-2a84b73423cc
            ],
            Server: [
                Kestrel
            ],
            Transfer-Encoding: [
                chunked
            ]
        },
        _createdNode: {
            Id: Guid_1,
            ParentId: Guid_2,
            ProjectId: specs-project,
            DisplayName: {
                en-GB: Test Node
            },
            Path: {
                en-GB: /en-gb/all-properties-node
            },
            ChildCount: 0,
            Version: {
                VersionNo: 0.1,
                CreatedBy: standard.user,
                Created: DateTimeOffset_1
            }
        }
    }
}
```

**Comment by SimonCropp** [comment, marked as answer]
try https://github.com/VerifyTests/Verify/blob/main/docs/serializer-settings.md#snippet-scrubinlineguids and use verifyhttp for better header scrubbing https://github.com/VerifyTests/Verify.Http#enable

**Comment by cpneale** [comment]
Thanks for the quick response.

This has helped with the GUIDs (i.e. all GUIDs contained in strings are scrubbed) but it has not scrubbed the DateTimes.

I now get the following.  I.e. the DateTime in the Date header is still present and there is also a DateTime in the HttpResponse content property. I had removed this previously to make my post more readable but I have included it here, truncated.  I suppose it depends on what the scrubbers recognise as a DateTime, as my values below are in two different formats.  Perhaps I need to create custom scrubbers for this but the example video I watched at least seemed to scrub the Date header in a http response.

Thanks in advance for any help.

```
{
  _response: {
    StatusCode: 201,
    Headers: {
      Date: [
        Tue, 16 Aug 2022 08:13:09 GMT
      ],
      Location: [
        http://localhost:5000/api/management/specs-project/nodes/Guid_1
      ],
      Server: [
        Kestrel
      ],
      Transfer-Encoding: [
        chunked
      ]
    },
    Content: {"version":{"versionNo":"0.1","createdBy":"standard.user","created":"2022-08-16T08:13:10.442Z"}
  },
  _createdNode: {
    Id: Guid_1,
    ParentId: Guid_2,
    ProjectId: specs-project,
    DisplayName: {
      en-GB: Test Node
    },
    Path: {
      en-GB: /en-gb/test-node
    },
   ChildCount: 0,
    Version: {
      VersionNo: 0.1,
      CreatedBy: standard.user,
      Created: DateTimeOffset_1
    }
  }
}

```

**Reply by SimonCropp** [reply]
can u add a `application/json`  ContentType header to the response and try again

**Comment by cpneale** [comment]
hi,

Yes, the ```application/json``` ContentType header works, thanks!
